packages/@cdktf/hcl2json/test/expressions.test.ts

```diff
@@ -1,13 +1,17 @@
 // Copyright (c) HashiCorp, Inc
 // SPDX-License-Identifier: MPL-2.0
-import { getReferencesInExpression } from "../lib";
+import { getReferencesInExpression, getExpressionAst } from "../lib";
 
 function parseExpression(expr: string) {
   return () =>
     expect(
       getReferencesInExpression("main.tf", expr)
     ).resolves.toMatchSnapshot();
 }
+function getAst(expr: string) {
```

Comment:
Maybe
```suggestion
function snapshotAst(expr: string) {
```

or

```suggestion
function testAst(expr: string) {
```
?

Reply:
But I do really like how you structured these tests!